api/utils/random.js

```diff
@@ -0,0 +1,10 @@
+const { v4: uuid } = require('uuid');
+
+function randomGenerator() {
+  const id = uuid();
+  return id.substr(0, 5);
```

Comment:
let's have integer OTPs only, people are stupid, software should be idiot proof.